handler/tag.go

```diff
@@ -53,3 +55,37 @@ func (t *Tag) CreateTag(c *gin.Context) {
 
 	c.JSON(http.StatusOK, tagPublic)
 }
+
+func (t *Tag) GetAllTags(c *gin.Context) {
+	tagPublicSlice, err := t.tagService.GetAllTags()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"message": "something happen in the server"})
+		return
+	}
+	c.JSON(http.StatusOK, tagPublicSlice)
+}
+
+func (t *Tag) GetTagById(c *gin.Context) {
+	tagIdStr, ok := c.Params.Get("id")
+	if !ok {
+		c.JSON(http.StatusBadRequest, gin.H{"message": "wrong body format"})
+		return
+	}
+	tagId, err := strconv.Atoi(tagIdStr)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"message": "wrong body format"})
+		return
```

Comment:
Please use the global `ErrBadRequestBody`.  You may need to rebase develop into your branch.